### ISSUE TITLE
Missed the 0.80V

### ISSUE THREAD

**Issue by hndrbrm** [raspberrypi/pico-sdk#1491]
https://github.com/raspberrypi/pico-sdk/blob/6a7db34ff63345a7badec79ebea3aaef1712f374/src/rp2_common/hardware_vreg/include/hardware/vreg.h#L26

I want to set the voltage to 0.80V but realize its not in there.

The datasheet says (Table 189. VREG Register):
`0000 to 0101 - 0.80V`

**Comment by lurch** [comment]
Yup, that's also what the comments in https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_regs/include/hardware/regs/vreg_and_chip_reset.h say. I guess this also means that the `VREG_VOLTAGE_MIN` in https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/hardware_vreg/include/hardware/vreg.h needs updating too.

**Comment by matsobdev** [comment]
You might not go anywhere with 0.80V anyway - probably won't boot.
